GUIDELINES.md

```diff
@@ -0,0 +1,104 @@
+# GUIDELINES REACT JS
+
+> Here you can find the Axa Front End Community recommandations for developping React Application
+
+- [GUIDELINES REACT JS](#guidelines-react-js)
+  - [Philosophy](#philosophy)
+  - [Organize your application](#organize-your-application)
+    - [General structure](#general-structure)
+    - [Naming convention](#naming-convention)
+  - [Stack :](#stack-)
+    - [ES Lint Configuration](#es-lint-configuration)
+    - [Recommanded Tools](#recommanded-tools)
+
+## Philosophy
+
+When you developping React application you **MUST** follow somes principles :
+
+- **DDD** : Grouping by feature. It is recommended to group the files by functionality, as the application evolves.
+- **Make it simple :** Starting application with simple components. Don't use abstractions, or complexe patterns until you need it.
+- **AHA Programming :** [Prefer duplication over the wrong abstraction](https://kentcdodds.com/blog/aha-programming).
+- **State :** Keep state as close to where it's needed as possible. Read [Gérer son état React](https://medium.com/@olivier.youf/g%C3%A9rer-l%C3%A9tat-de-son-application-react-d160210ce68c)
+- **Functionnal programming :** Fonctions pures et prog fonctionnelle => TODO JOHN
+
+## Organize your application
+
+### General structure
+
+    shared
+        components
+            SharedComponent
+        helpers
+        images
+        styles
+    layout
+        App
+        Header
+        Footer
+        Routes
+    pages
+        MyComponent1
+        MyComponent2
+
+- **shared :** All shared components/functions.
+- **layout :** Contains general components of the application like header, footer, menu, etc.
+- **pages :** Contains content of pages.
+
+### Naming convention
+
+- Uppercase for component files name (except index.js)
```

Comment:
- PascalCase for component files name (except index.js)

Reply:
yes